test/quantization/test_quantize_script.py

```diff
@@ -932,16 +932,19 @@ def __init__(self):
                 self.avgpool = torch.nn.AdaptiveAvgPool2d((1, 1))
 
             def forward(self, x):
-                x = torch.dequantize(x)
+                # type: (List[Tensor]) -> Tensor
+                x, y = torch.dequantize(x)
                 x = self.maxpool(x)
-                x = self.avgpool(x)
-                return x
+                y = self.avgpool(y)
+                return x + y
         x = torch.randn([1, 3, 10, 10], dtype=torch.float)
         x = torch.quantize_per_tensor(x, 0.5, 1, torch.quint8)
+        input = [x, x]
```

Comment:
Do both quantize_per_tensor and dequantize now support list of tensors?

Reply:
yeah that's correct: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L3580, but I think we don't really need the list version of quantize_per_tensor right now